modin/experimental/backends/omnisci/query_compiler.py

```diff
@@ -81,9 +82,24 @@ def build_method_wrapper(name, method):
 
     @wraps(method)
     def method_wrapper(self, *args, **kwargs):
+        # If the method wasn't found in the 'BaseQueryCompiler' that means,
+        # that we're calling one that is OmniSci backend-specific, if we intend
+        # to fallback to pandas on 'NotImplementedError' then the call of this
+        # private method is the cause of some public QC method, that will catch
+        # arisen here exception and do fallback properly
```

Comment:
```suggestion
        # private method is caused by some public QC method, so we catch
        # the exception here and do fallback properly
```
I hope I understood what you meant here, so I tried to correct the wording according to my understanding